java8/src/test/java/io/cucumber/java8/TypeDefinitionsStepdefs.java

```diff
@@ -0,0 +1,90 @@
+package io.cucumber.java8;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import io.cucumber.datatable.DataTable;
+
+import java.awt.Point;
```

Comment:
Looking forwards to Java 12+ it might be good to define a tuple of integers ourselves. That way we won't have to import the whole of awt.